app/controllers/employee.controller.js

```diff
@@ -0,0 +1,43 @@
+const EmployeeModel = require('../models/employee.model');
+
+const createEmployee = async (employeeData) => {
+
+    const {
+        departmentID,
+        employeeName,
+        employeeEmail,
+        employeeAge,
+        employeePosition
+    } = employeeData;
+
+    const newEmployee = new EmployeeModel({
+        departmentID: departmentID,
+        employeeName: employeeName,
+        employeeEmail: employeeEmail || `${empName} does not have email`,
```

Comment:
I don't understand, why you use it here, if email is not require, just leave it as a empty field